server/data-migrations/1667076251897-BackfillDataSources.ts

```diff
@@ -0,0 +1,46 @@
+import { DataQuery } from 'src/entities/data_query.entity';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class BackfillDataSources1667076251897 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const entityManager = queryRunner.manager;
+
+    const versions = await entityManager
+      .createQueryBuilder()
+      .select()
+      .from('app_versions', 'app_versions')
+      .getRawMany();
+
+    for (const version of versions) {
+      let runjsDS, restapiDS;
+      for await (const kind of ['runjs', 'restapi']) {
+        const dataSourceResult = await entityManager.query(
+          'insert into data_sources (name, kind, app_version_id, app_id) values ($1, $2, $3, $4) returning "id"',
+          [`${kind}default`, `${kind}default`, version.id, version.app_id]
```

Comment:
We can give any name, won't be visible anywhere in front end, but kind should not be `restapi`, since its already used and we can't differentiate between rest api queries having datasource and not. Named it in such a way that to identify they are default one, should not be visible or editable for users

Reply:
Yes, we are managing the same in app imports and migrations